### ISSUE TITLE
SteamVR crash when using OpenVr Input Emulator

### ISSUE THREAD

**Issue by overdrive-ostrich** [GPUOpen-LibrariesAndSDKs/Radeon-ReLive-VR#76]
Because SteamVR doesn't see my Quest as "native," when I run games on ReLive, I have to use [OpenVR Input Emulator](https://github.com/matzman666/OpenVR-InputEmulator). This is because my controllers need to be set to Vive controllers in some games, and I need a 30° offset to compensate.
This setup worked fine until IVRServerDriverHost got updated to version 006 in the driver DLLs on Sept 17 as described in [this thread](https://github.com/matzman666/OpenVR-InputEmulator/issues/198). A hotfix as created on their end, but now the combination of that and ReLive doesn't work. Both programs work fine when run individually, but when I go to boot them together, SteamVR crashes as soon as my Quest connects.

**Comment by GennadiyAMD** [comment]
@overdrive-ostrich - have you tried just setting the headset emulation to HTC Vive in Radeon Settings? This will work because the OpenVR driver doesn't really care what the headset is (Quest of Vive Focus+) and will try to remap buttons on the controller to match as much as possible. Buttons will work, the only weird thing is emulating a trackpad with a joystick, it's awkward, but it does work and I guess it would be equally awkward with Input Emulator.

**Comment by overdrive-ostrich** [comment]
@GennadiyAMD yes, that works just fine (though I usually have to manually remap the buttons), but the issue is that the Vive controllers are more like a wand, whereas the Oculus Touches are like a pistol-grip. This means I have to point way lower (30 degrees), like I have to point at their knees to shoot them in the chest. I have the problem with Arizona Sunshine, Boneworks, and The Walking Dead.

**Comment by nocxtious** [comment]
I have also run into this problem with Boneworks specifically and tried to fix it with the input emulator and got the crashing as well.  Is there a way we can manually adjust the controller offset or something similar to this: https://steamcommunity.com/sharedfiles/filedetails/?id=1980632821  (I know its for ALVR but I wasn't sure if something like that would also be possible with ReLive, I have yet to try ALVR and would like to stick with ReLive since it works so well for everything else)

**Comment by overdrive-ostrich** [comment]
@nocxtious that would be fantastic! I don't have the technical knowledge to do something like that though. Maybe someone who can will be liked to/stumble across this thread
